src/main/java/com/mitchej123/hodgepodge/mixins/late/bibliowood/forestry/MixinTabRegistry.java

```diff
@@ -0,0 +1,49 @@
+package com.mitchej123.hodgepodge.mixins.late.bibliowood.forestry;
+
+import net.minecraft.block.Block;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.ModifyVariable;
+
+import cpw.mods.fml.common.registry.GameRegistry;
+import jds.bibliowood.forestrywood.WoodsLoader;
+
+@Mixin(WoodsLoader.class)
+public class MixinTabRegistry {
+
+    /**
+     * @author DrParadox7
+     * @reason Bibliowoods references multiple Forestry ID planks that no longer exists due to them all being unified
+     *         into 1 block.
+     */
+    @ModifyVariable(method = "initRecipes", at = @At(value = "STORE", ordinal = 0), remap = false)
```

Comment:
```suggestion
    @ModifyVariable(method = "initRecipes", at = @At(value = "STORE"), ordinal = 0, remap = false)
```
Fix the `ordinal`, for consistency.